wagtail/rich_text/__init__.py

```diff
@@ -99,6 +99,11 @@ def __str__(self):
     def __bool__(self):
         return bool(self.source)
 
+    def __eq__(self, other):
+        if isinstance(other, RichText):
+            return self.source == other.source
+        return False    
```

Comment:
Remove trailing whitespace

```suggestion
        return False
```